plugins/exporter.koplugin/main.lua

```diff
@@ -57,6 +57,7 @@ local function migrateSettings()
         new_settings["joplin"].port = settings.joplin_port
         new_settings["joplin"].token = settings.joplin_token
         new_settings["readwise"].token = settings.readwise_token
+        new_settings["memos"].api = settings.memos_api
```

Comment:
It isn't required. It is just for previous iteration of the plugin. Please remove